crates/forge/bin/cmd/inspect.rs

```diff
@@ -169,8 +169,7 @@ impl InspectArgs {
                     for ev in abi.events.iter().flat_map(|(_, events)| events) {
                         let types = ev.inputs.iter().map(|p| p.ty.clone()).collect::<Vec<_>>();
                         let sig = &ev.signature();
-                        let topic =
-                            hex::encode(&keccak256(sig.strip_prefix("0x").unwrap_or(sig))[..4]);
+                        let topic = hex::encode(&keccak256(sig.strip_prefix("0x").unwrap_or(sig)));
```

Comment:
I think having the `0x` prefix here being included makes sense given it is a requirement for topics / use in assembly